.github/workflows/build_and_test_dbt.yaml

```diff
@@ -62,14 +62,14 @@ jobs:
           if [[ $MODIFIED_RESOURCES_ONLY == 'true' ]]; then
             if [[ $MANUALLY_DISPATCHED == 'true' ]]; then
               echo "Running build on manually selected resources"
-              dbt run -t "$TARGET" -s ${{ inputs.models }} --defer --state "$STATE_DIR"
+              dbt build -t "$TARGET" -s ${{ inputs.models }} --defer --state "$STATE_DIR" --resource-types model seed
```

Comment:
[`dbt run`](https://docs.getdbt.com/reference/commands/run) only builds models, but now that our DAG includes seeds, we need to switch to [`dbt build`](https://docs.getdbt.com/reference/commands/build) instead. Note that the `--resource-types` flag is quite important for us, since our DAG is big enough that we don't want to run tests when we rebuild resources.